grants_tagger/tag_grants.py

```diff
@@ -15,7 +15,7 @@ def yield_grants(grants_path):
         for grant in csv_reader:
             yield grant
 
-def tag_grants(grant):
+def tag_grants(grant, threshold):
```

Comment:
is this going to break anything by not putting in a default value? (i.e. is tag_grants being used anywhere else without the threshold argument)

Reply:
not but good point, for backwards compatibility i should add a default